test/src/tests/generated/lro/models.go

```diff
@@ -2650,7 +2650,7 @@ func (sp *SubProduct) UnmarshalJSON(body []byte) error {
 type SubProductProperties struct {
 	ProvisioningState *string `json:"provisioningState,omitempty"`
 	// ProvisioningStateValues - Possible values include: 'ProvisioningStateValues1Succeeded', 'ProvisioningStateValues1Failed', 'ProvisioningStateValues1Canceled', 'ProvisioningStateValues1Accepted', 'ProvisioningStateValues1Creating', 'ProvisioningStateValues1Created', 'ProvisioningStateValues1Updating', 'ProvisioningStateValues1Updated', 'ProvisioningStateValues1Deleting', 'ProvisioningStateValues1Deleted', 'ProvisioningStateValues1OK'
-	ProvisioningStateValues ProvisioningStateValues `json:"provisioningStateValues,omitempty"`
+	ProvisioningStateValues ProvisioningStateValues1 `json:"provisioningStateValues,omitempty"`
```

Comment:
>ProvisioningStateValues1 [](start = 25, length = 24)

Little side effect which is not that cool

Reply:
previous value was actually incorrect, this is the correct value if you look at the comment of the field